### ISSUE TITLE
pac-container dont work

### ISSUE THREAD

**Issue by Kintabor** [ubilabs/geocomplete#276]
Hey everyone, today the dropdown-popup doesnt work anymore (the list with the search-suggestions).
Didnt know why, but I see this at the console:

"NetworkError: 404 Not Found - https://encrypted-tbn0.gstatic.com/favicon?q=tbn:ANd9GcT6jdI45NEKdPWqP1PWenN--DPwsKL-ylYNDreDRElqZ7d8PKQiD5NM3AlyKXW_taK6pKZgPRS5m_ZEdGGsRtio_mqYtH0Ao_KP-aPoniKw4g61n3kUtDMzp8VLAP97-Jvv22eUwrIist_QPedQVji1P-EADkoAhhV7tS7zs8R0n3YJ3zGaqIG04giR"

Anyone a hint?

Greetz,

**Comment by eljoncurry2** [comment]
This has nothing to do with this plugin not working.  This is simply a 404 (not found) error for a favicon or image.

**Comment by Kintabor** [comment]
You're right. The problem is the places autocomplete, it doesnt work since I switch to https. Is there a workarround?
